### DISCUSSION TITLE
Unhandled write error

### DISCUSSION THREAD

**Discussion by ghubz** [WeblateOrg/weblate#6178] [General]
In sentry's integration I get 

> GET 'lock:lock:gitlock'
> {
> redis.command: GET, 
> redis.key: lock:lock:gitlock
> }
> 
> SCRIPT LOAD b'\n    if redis.call("get", KEYS[1]) ~= ARGV[1] then\n        return 1\n    else\n        redis.call("del", KEYS[2])\n        redis.call("lpush", KEYS[2], 1)\n        redis.call("pexpire", KEYS[2], ARGV[2])\n        redis.call("del", KEYS[1])\n        return 0\n    end\n'
> {
> redis.command: SCRIPT LOAD
> }
> 
> EVALSHA '90da91872223f85eb1376a16e3c4df22ace8c54b' 2 'lock:lock:gitlock' 'lock-signal:lock:gitlock' '2sqzjaGX3l41CVnJF36RbnnO' 1000
> {
> redis.command: EVALSHA
> }

Any ideas?

**Comment by nijel** [comment]
What is the actual traceback for the error? This one looks like regular locking operation.
